command/git.go

```diff
@@ -34,7 +34,12 @@ type gitFetchProject struct {
 	// Note: If a module does not have a revision it will use the module's branch to get the project.
 	Revisions map[string]string `plugin:"expand"`
 
-	Token string `plugin:"expand"`
+	// ProjectToken is sourced from the project's YAML file
+	ProjectToken string `plugin:"expand" mapstructure:"token"`
+	// GlobalGitHubOauthToken is sourced from db.admin.find({"_id": "global"},{"credentials.github": 1})
+	GlobalGitHubOauthToken string `mapstructure:"global_github_oauth_token"`
```

Comment:
This shouldn't be a field in the struct. I think the approach should be to set alphaToken from the expansion directly if not set already